packages/datadog-plugin-jest/src/index.js

```diff
@@ -256,5 +416,15 @@ module.exports = [
       this.unwrap(JsdomEnvironment.prototype, 'teardown')
       JsdomEnvironment.prototype.handleTestEvent = JsdomEnvironment.prototype.handleTestEvent._dd_original
     }
+  },
+  {
+    name: 'jest-jasmine2',
```

Comment:
Any way `jest` could be instrumented in a way that doesn't require a lot of new code for each sub-package?

Reply:
`jest` is a library to put together many blocks, more than anything. Every test runner does its own thing (different folder and file structures), so if we want test visibility, I don't think there's any workaround. 
It might be an error to talk about "jest" compatibility though. We might want to say we support `jest-circus` and `jest-jasmine2`